duckduckgo/iOS
# Rebuild breakage form with SwiftUI

Task/Issue URL: https://app.asana.com/0/inbox/1164476816701138/1203944813756141/1204945810392144
Tech Design URL: https://app.asana.com/0/481882893211075/1204939034484684/f
CC: @brindy 

**Description**:
This PR rebuilds the breakage form with SwiftUI which accomplishes two things:
1. Add a dropdown for breakage category to match other platforms
2. Add a description field so users can add additional information



**Steps to test this PR**:
1. Tapping the report button from the dashboard and browsing menu should open the new form
3. Select a category
4. Entering a description should be optional to submitting the form
5. The bar button and blue submit button should submit the report and dismiss the view
6. Cancel should dismiss the view without submitting



**Copy Testing**:


**Orientation Testing**:


**Device Testing**:


**OS Testing**:


**Theme Testing**:


---
###### Internal references:
[Software Engineering Expectations](https://app.asana.com/0/59792373528535/199064865822552)
[Technical Design Template](https://app.asana.com/0/59792373528535/184709971311943)


---
- To see the specific tasks where the Asana app for GitHub is being used, see below:
  - https://app.asana.com/0/0/1203944813756141
